pages/TheExplorationGallery.html

```diff
@@ -11,32 +11,39 @@
   <p>
     After signing in to Oppia, you'll end up on the Home page. Even before you
     start creating your own explorations, there are a number of things you can
-    do. You can click on the icon at the top left corner of the page to access
-    the blog, join the discussion forum or access the Participation Playbook.
-    You can also check out a variety of existing explorations in different
-    categories, or create one of your own by clicking on "Create Exploration"
+    do. You can click on the about icon at the top right corner of the page 
+    (for large screen sizes) or the icon at the top left corner (for small 
+    screen sizes) to access the blog, join the discussion forum or access other pages on the site. You can also check out a variety of existing explorations 
+    in different categories, or create one of your own by clicking on "Create Exploration" 
     at the top right.
   </p>
 
-  <img src="static/images/frontPage.png" width="100%">
+  <img src="static/images/frontPageLargeScreen.png" width="100%">
 
   <p>
-    Below the banner, the homepage contains a list of all published, non-draft
-    explorations on the server. If there are a lot of explorations, not all of
-    them will be shown at once. As you scroll down, more will be loaded.
+  	for other screen sizes
   </p>
 
-  <img src="static/images/gallery.png" width="100%">
+  <img src="static/images/frontPageSmallScreen.jpg" width="100%">
+
+  <p>
+    The <b> library </b> contains a list of all published, non-draft explorations
+    on the server. The explorations are grouped in various categories that 
+    they best fit in. For example, the quadratic equation is found under the 
+    category "Mathematematics and Statistics".
+  </p>
+
+  <img src="static/images/Library.png" width="100%">
 
   <p>
     You can filter the list by searching for specific topics, or selecting
     particular categories and languages. (You can also change the default
     language settings on your Preferences page.) In the following screenshot,
-    the user has selected the "Mathematics" category (circled in orange), so
-    the gallery shows only mathematics explorations.
+    the user has selected the "Mathematics and statistics" category, so
+    the gallery shows only mathematics and statistics explorations.
   </p>
 
-  <img src="static/images/galleryCategoryHighlighted.png" width="100%">
+  <img src="static/images/libraryCategoryHighlighted.png" width="100%">
 
   <p>
     To play an exploration, click anywhere on its gallery tile. If you have
```

Comment:
gallery tile --> summary tile, here and elsewhere. Drop the next sentence about 'Edit' -- this no longer appears.